.github/CODEOWNERS

```diff
@@ -0,0 +1,21 @@
+# CODEOWNERS adds reviewers depending on the location of the change.
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies JS files, only @js-owner and not the global
+# owner(s) will be requested for a review.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @forcedotcom/pdt will be requested for
+# review when someone opens a pull request.
+*       @forcedotcom/pdt
+
+# If the change occurs within salesforcedx-vscode-lwc then add the Lightning Tooling team.
+/packages/salesforcedx-vscode-lwc @forcedotcom/lightning-tooling
+
+# If the change occurs within salesforcedx-vscode-lightning then add the Lightning Tooling team.
+/packages/salesforcedx-vscode-lightning @forcedotcom/lightning-tooling
+
+# If the change occurs within the docs directory, add the doc maintainers.
+/docs/ @forcedotcom/doc-maintainers
```

Comment:
We may need to change the sub-team name from doc-maintainers to something like 'pdt-doc-maintainers'.

Reply:
For my understanding, where are the members of a team, say "@forcedotcom/pdt" defined ?

Reply:
Nevermind. I see where the teams are.

Reply:
Good question, in case anyone else isn't sure here is the link:
https://github.com/orgs/forcedotcom/teams/tooling-team

Luis added the teams 'PDT' and 'Doc Maintainers'.